unified-codes/libs/ui/src/lib/components/inputs/molecules/ToggleButton.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import ToggleButton from './ToggleButton';
+
+export default {
+  component: ToggleButton,
+  title: 'ToggleButton',
+};
+
+export const Unselected = () => {
+  return <ToggleButton>Toggle Button</ToggleButton>;
+};
```

Comment:
```suggestion
export const Unselected = () => (
  <ToggleButton>Toggle Button</ToggleButton>
);
```

an alternative, with a few less characters required

Reply:
Switched it over with 1169b59, nice to save those chars 😸